test/screenshot/tab/index.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import Tab, {TabProps} from '../../../packages/tab';
+// https://github.com/material-components/material-components-web-react/issues/513
+// @ts-ignore
+import MaterialIcon from '../../../packages/material-icon/index';
+import '../../../packages/tab-indicator/index.scss';
+import '../../../packages/tab/index.scss';
+import './index.scss';
+
+interface TabContentType {
+  num: number;
+}
+
+const TabContent: (args: TabContentType) => React.ReactElement<any> = ({
+  num, // eslint-disable-line react/prop-types
+}) => (
+  <React.Fragment>
+    <MaterialIcon className='mdc-tab__icon' icon='favorite' />
+    <span className='mdc-tab__text-label'>Tab {num}</span>
+  </React.Fragment>
+);
+
+interface TabsProps {
+  children: React.ReactNode;
+}
+
+function isElementTab(element: any): element is Tab {
```

Comment:
This is a bad type guard.  I can call `isElementTab(5)` and `5` would be treated like a `Tab`.

You should make the check more specific, or rename the function to `isNotNull`.

Reply:
so it would be like this:

```
function isNotNull(element: any): element is Tab {
```

That wouldn't look too good either?